jobs/deploy-internal-service.groovy

```diff
@@ -13,61 +13,70 @@ import org.khanacademy.Setup;
 // The easiest setup ever! -- we just use the defaults.
 new Setup(steps
 
+).addChoiceParam(
+        "SERVICE",
+        ["buildmaster"],
+        """<b>REQUIRED</b>. The service to deploy, such as "buildmaster" or "ingress".""",
 ).addStringParam(
-    "SERVICE",
-    """<b>REQUIRED</b>. The service to deploy, such as "buildmaster" or
-    "ingress".""",
-    ""
-
+        "BRANCH",
+        """<b>REQUIRED</b>. The branch to deploy from. Typically "master".""",
+        "master"
+).addChoiceParam(
+        "GCLOUD_PROJECT",
+        ["khan-test", "khan-internal-services"],
```

Comment:
It might be good to limit this to just khan-test for now. If I understand correctly, running this against branch: master and project: khan-internal-services will likely result in a broken buildmaster.